installer/cape2.sh

```diff
@@ -1187,7 +1187,8 @@ function install_CAPE() {
       crudini --set conf/reporting.conf runstatistics enabled yes
     fi
 
-    python3 utils/community.py -waf -cr --url "https://github.com/polyswarm/CAPESandbox-community/archive/develop.tar.gz"
+    # pin the signatures to a specific hash with our fixes, until they are merged upstream
+    python3 utils/community.py -waf -cr --url "https://github.com/polyswarm/CAPESandbox-community/archive/7e530cc6d28c692528d236ba3efb73f62b3221e1.tar.gz"
```

Comment:
I do not understand this change, can't we keep installing from the branch https://github.com/polyswarm/CAPESandbox-community/tree/develop and just merge our fixes there, since it is our repo?

Reply:
Yes we can still do that.  This was helping me develop because, without making a change in the this submodule it was using a cached docker layer and not really updating the community repo.

Reply:
It seems to me you got the HEAD of the develop branch here already, so I don't see why we can't install directly from develop

Reply:
Ah, ok, I would rather keep the installing from develop then, and just add some cache invalidation feature (such as introducing an `ARG`) so that we can force the rebuild. Alternatively, you can explicitly set the sha you want to install as an env var, but I would prefer not to have it hardcoded.